modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx

```diff
@@ -248,6 +273,13 @@ export default function EditNotificationTemplate({
 			[defaultLanguageId]: '',
 		},
 		description: '',
+		editorType: 'richText' as editorTypeOptions,
+		freemarkerTemplate: {
+			lineCount: 0,
+			template: `<#--
+		Add elements from the sidebar to define your template. Type "\${" to use the autocomplete feature
+		-->`,
```

Comment:
Hey @naraandrade, I think you forgot to use the language key here, in this case Brian prefers to pass this `"\${"` as a parameter of a language key, for that you can use a `Liferay.Util.sub`, like:

```tsx
`<#-- ${Liferay.Util.sub(
	Liferay.Language.get(
		'add-elements-from-the-sidebar-to-define-your-template-type-x-to-use-the-autocomplete-feature
	),
	["\${"]
)} -->`

```

Reply:
Thank you for pointing this out, Daniel ^^